extensions/amp-analytics/0.1/vendors/blackcrowai.json

```diff
@@ -0,0 +1,27 @@
+{
+  "requests": {
+    "pageview": "https://api.blackcrow.ai/v1/js/events/view"
+  },
+  "triggers": {
+    "trackPageview": {
+      "on": "visible",
+      "request": "pageview"
+    }
+  },
+  "transport": {
+    "beacon": true,
```

Comment:
default value is `true`. only need to specify `useBody` here.

Reply:
Great, updated, thanks! 👍